docs/generated/actions/run_tests.md

```diff
@@ -192,6 +192,18 @@ run_tests(
 )
 ```
 
+```ruby
+# run tests on a pure swift package (SPM), no xcodeproj or xcworkspace required
+run_tests(
+  package_path: ".",
+  scheme: "MySwiftCode-Package", # Must have -Package
+  clean: true,
+  device: "iPhone 11 Pro", # Required for destination
+  result_bundle: true, # Last two lines may be required for proper xcresults output
+  output_directory: Dir.pwd + "/test_output"
```

Comment:
Perhaps this would be better formatted like this?

```suggestion
  # These last two lines may be required for proper .xcresult output:
  result_bundle: true, 
  output_directory: Dir.pwd + "/test_output"
```